doc/contributing.rst

```diff
@@ -1,221 +1,760 @@
-:orphan:
+.. contents:: Page contents
+   :local:
+   :depth: 2
 
-.. include:: links.inc
+.. _contributing:
 
-.. _contribute_to_mne:
+.. include:: ../CONTRIBUTING.rst
 
-How to contribute to MNE-python
-===============================
 
-.. contents:: Contents
-   :local:
-   :depth: 1
+Overview of contribution process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We are open to many types of contributions, from bugfixes to functionality
-enhancements. mne-python_ is meant to be maintained by a community of labs,
-and as such, we seek enhancements that will likely benefit a large proportion
-of the users who use the package.
+In general you'll be working with three different copies of the MNE-Python
+codebase: the official copy at https://github.com/mne-tools/mne-python (usually
+called "upstream"), your `fork`_ of the upstream repository (similar URL, but
+with your username in place of ``mne-tools``, and usually called "origin"), and
```

Comment:
I'd put `origin` and `upstream` in double backticks so they mentally and visually link nicely to their usages below